CONTRIBUTING.md

```diff
@@ -1,4 +1,37 @@
 # Contributing guidelines
 
-This project is in its very early stages and is not actively soliciting
-contributions at the present time.
+If you're looking for something to work on, please look at our 
+[milestones](https://github.com/kubernetes/cluster-registry/milestones) or our
+[open issues](https://github.com/kubernetes/cluster-registry/issues)
+and assign an issue to yourself.
```

Comment:
Self-assign will only be supported for people that are already in the kubernetes org (silly limitation of github IMO).  Should we keep this language?

Reply:
I've added some more details and a link to the k8s community membership docs. WDYT?